sync/etcdv3/etcd_test.go

```diff
@@ -285,6 +291,98 @@ func TestWatchWithRevision(t *testing.T) {
 
 }
 
+func TestFetchMultipleNodes(t *testing.T) {
+	sync := newSync(t)
+	sync.etcdClient.Delete(context.Background(), "/", etcd.WithPrefix())
+
+	err := sync.Update("/path/to/somewhere", "test1")
+	if err != nil {
+		t.Fatalf("unexpected error")
+	}
+	err = sync.Update("/path/to/elsewhere", "test2")
+	if err != nil {
+		t.Fatalf("unexpected error")
+	}
+	err = sync.Update("/path/notto/elsewhere", "test3")
+	if err != nil {
+		t.Fatalf("unexpected error")
+	}
```

Comment:
maybe better to add `/pathnottobeincluded` as well to confirm it's not included in the output.